doc/nrf/releases_and_maturity/releases/release-notes-3.0.0-preview1.rst

```diff
@@ -91,8 +91,6 @@ Developing with Front-End Modules
 
 * Added support for the following:
 
-  * :ref:`nRF2220 Front-End Module <ug_radio_fem_nrf2220>`.
```

Comment:
We could also remove the entry about "The temperature compensation feature for the nRF2220 Front-End Module." - https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-3.1.0.html#developing_with_front-end_modules from 3.1.0, 3.1.0 - preview 1,2,3 release notes.

Reply:
Done